matisse/src/main/java/com/zhihu/matisse/engine/impl/GlideEngine.java

```diff
@@ -32,49 +33,47 @@
 
     @Override
     public void loadThumbnail(Context context, int resize, Drawable placeholder, ImageView imageView, Uri uri) {
+        RequestOptions options = new RequestOptions().placeholder(placeholder).override(resize, resize).centerCrop();
         Glide.with(context)
-                .load(uri)
-                .asBitmap()  // some .jpeg files are actually gif
-                .placeholder(placeholder)
-                .override(resize, resize)
-                .centerCrop()
-                .into(imageView);
+            .asBitmap()  // some .jpeg files are actually gif
+            .load(uri)
+            .apply(options)
+            .into(imageView);
```

Comment:
Actually you are using a 3.7.0 Glide，how can you use 4.x API here?

Reply:
I am using RequestOptions object here which is added in Glide v4.0 http://bumptech.github.io/glide/doc/options.html#requestoptions. Look at line number [36](https://github.com/thakkaryash94/Matisse/blob/develop/matisse/src/main/java/com/zhihu/matisse/engine/impl/GlideEngine.java#L36)